plugins/rapid7_insightvm/komand_rapid7_insightvm/util/update_shared_credential_util.py

```diff
@@ -0,0 +1,296 @@
+from typing import Union, Tuple, Dict
+
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+from komand_rapid7_insightvm.util.util import check_not_null
+
+
+def ssh_setup(account: Dict[str, any]) -> Tuple[str, str, str]:
```

Comment:
I suggest changing `any` to `Any` from `typing` library
 😄 
```suggestion
def ssh_setup(account: Dict[str, Any]) -> Tuple[str, str, str]:
```